rkt/rkt.go

```diff
@@ -165,7 +216,7 @@ func garbageDir() string {
 }
 
 func getKeystore() *keystore.Keystore {
-	if globalFlags.InsecureSkipVerify {
+	if globalFlags.InsecureFlags.SkipImageCheck() {
```

Comment:
Hm, actually we [do use that fact](https://github.com/coreos/rkt/blob/99aba4317982dea23c92f1baf83147e2839bf6a9/rkt/images.go#L474) so this is fine.